README.md

```diff
@@ -2,49 +2,103 @@
 
 [![License][License-Image]][License-Url]
 
-This is the Git repo of the official Docker image for [nats]. See the Hub page
-for the full readme on how to use this Docker image and for information
-regarding contributing and issues.
+This is the repo for building the official [NATS server Docker images]. If you
+just want to use NATS server, then head over to [Docker Hub]. You don't need
```

Comment:
fix link: `[Docker Hub](https://hub.docker.com/_/nats)`

Reply:
Ah, that should work since I've specified the link down below.

```
[Docker Hub]: https://hub.docker.com/_/nats
```

I hope. Is it broken on your end? :grimacing:

Reply:
Oh I see... sorry was doing dumb code review and should have checked the "View File" to see that the links is ok. Please ignore.